vscode/rootfs/var/lib/code-server/defaults/core/example-mqtt.sh

```
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# shellcheck source=./PATHS.sh
+source "/etc/s6-overlay/s6-rc.d/paths.sh"
+
+# Vars
+ENABLE=false
+PULL=true
+openMQTT_git=https://github.com/1technophile/OpenMQTTGateway.git
+openmqtt_src_files="$USER_CUSTOM_DIR"/OpenMQTTGateway
```

Shellcheck (0.10.0)

[warning] USER_CUSTOM_DIR is referenced but not assigned.

(SC2154)